pkg/utils/taints.go

```diff
@@ -0,0 +1,121 @@
+package utils
+
+// Copy paste from SNR - https://github.com/medik8s/self-node-remediation/blob/main/pkg/utils/taints.go
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	farV1alpha1 "github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+)
+
+var (
+	loggerTaint           = ctrl.Log.WithName("taints")
+	FARNodeNoExecuteTaint = &corev1.Taint{
+		Key:    farV1alpha1.Medik8sNoExecuteTaintKey,
+		Value:  farV1alpha1.FARNoExecuteTaintKeyValue,
```

Comment:
I think the value can be removed, since we have a unique key and always using the same value

Reply:
Wouldn't it be useful for identifing the remediatior by the tain't value?

Reply:
The key is already unique for the remediator, IIUC it would have made sense if the same remediator would like to apply the same taint twice with different distinguished variations.

Reply:
> IIUC it would have made sense if the same remediator would like to apply the same taint twice with different distinguished variations.

Could be, but SNR doesn't have this scenario, and it's taint does have a value - [SNR's remediation taint](https://github.com/medik8s/self-node-remediation/blob/main/controllers/selfnoderemediation_controller.go#L59-L63).

I was thinking on a different case with NHC escalation.
When different remediators are running one after another, and the taint's value can act as an identifier for the remediator that it is running ATM. I assume that it can easily be found/described in NHC log, but maybe adding in the taint's value will be useful for other stuff (e.g. fetching metrics on the acting remediator with NHC)

Reply:
> The key is already unique for the remediator

Looking at SNR, I´d say no, it isn't: https://github.com/medik8s/self-node-remediation/blob/main/controllers/selfnoderemediation_controller.go#L59

Using the value for the remediator makes sense to me